zulip/integrations/slack/slackdata2zulipdata.py

```diff
@@ -139,21 +141,32 @@ def channels2zerver_stream(slack_dir, realm_id, added_users):
             id=stream_id_count)
         zerver_stream.append(stream)
         added_channels[stream['name']] = stream_id_count
+       
+        # construct the recipient object and append it zerver_recipient
+        recipient = dict(
```

Comment:
With this, L179-L183 becomes redundant.

Reply:
To be precise, https://github.com/rht/python-zulip-api/blob/d277eec2d57ea25fc6131fdc03194f216f575048/zulip/integrations/slack/slackdata2zulipdata.py#L166-L170